js/extras/util.js

```diff
@@ -253,7 +253,7 @@ function util() {
             arrowDom.style.top = (top + height - offsetTop) + 'px';
 
             if (sortType == "alternatedimension" && chartType == "horizontal") {
-                arrowDom.style.top = height / 2 + tipHeight / 2 + "px";
```

Comment:
can u create a new function for this calculation 
there are following advantages
1)js file will be redunce..easy to read the js file
2)this func can be reutilized

Reply:
ok

Reply:
@johibkhan2 
this function is already created for the calculation of the tooltip position.

Reply:
so replace these lines of code with function